### DISCUSSION TITLE
[Android][Auth]Not found FirebaseAuth.useEmulator(String, int)

### DISCUSSION THREAD

**Discussion by woodnynobu** [invertase/react-native-firebase#4824] [Q&A]
Please tell me how to resolve not found firebaseAuth.useEmulator()
It is occurring on android, iOS build success.
Should I change FirebaseSDK BOM version?

As reference information
ReactNative Auth:10.0.0
Android FirebaseSDK BOM:18.0.0（build failed)
iOS FirebaseSDK:7.0.0（build success）

Package:io.invertase.firebase.auth
File:ReatNativeFIrebaseAuthModule.java
```
@ReactMethod
public void useEmulator(String appName, String host, int port) {
    Log.d(TAG, "useEmulator");
    FirebaseApp firebaseApp = FirebaseApp.getInstance(appName);
    FirebaseAuth firebaseAuth = FirebaseAuth.getInstance(firebaseApp);
    firebaseAuth.useEmulator(host, post); →useEmulator() not found!
}
```

**Comment by mikehardy** [comment]
If you use a BOM as old as 18, it won't have the symbol or implementation. Auth.useEmulator was introduced in BOM 26, and current is 26.3.0. Update your BOM and your build will work.

**Reply by woodnynobu** [reply]
I solved it by changing the version of react-native-firebase/app to 9.0.0.
Thank you for your answer.

**Reply by mikehardy** [reply]
Not sure why you'd do that, you updated a version but not to current, why not? We are on 10.5.x here now - it's generally good practice to stay up to date, the underlying SDKs move very quickly and it's the easiest way to track them

**Reply by walshbm15** [reply]
Hey, I know this is a few years old, but stumbled across this as I am having a similar issue. I am trying to get a local Quarkus app to connect to the Firebase Auth emulator. I have the most recent bom included in my project but this method is still not available to me. Showing my dependency list below.

```
    <dependencies>
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-arc</artifactId>
        </dependency>
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-junit5</artifactId>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>io.rest-assured</groupId>
            <artifactId>rest-assured</artifactId>
            <scope>test</scope>
        </dependency>

        <!-- Quarkus dependencies -->
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-resteasy</artifactId>
        </dependency>
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-resteasy-jackson</artifactId>
        </dependency>
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-security</artifactId>
        </dependency>
        <dependency>
            <groupId>com.google.firebase</groupId>
            <artifactId>firebase-bom</artifactId>
            <version>33.0.0</version>
            <type>pom</type>
        </dependency>

        <!-- Firebase Admin SDK -->
        <dependency>
            <groupId>io.quarkiverse.googlecloudservices</groupId>
            <artifactId>quarkus-google-cloud-firebase-admin</artifactId>
            <version>2.9.0</version>
        </dependency>
    </dependencies>```
